include/util.h

```diff
@@ -18,33 +18,35 @@ class Timer {
     bool gpu_;
     std::chrono::time_point<std::chrono::high_resolution_clock> start_time_;
     std::chrono::time_point<std::chrono::high_resolution_clock> stop_time_;
-    CudaEvent start_event_;
-    CudaEvent end_event_;
+    CudaEvent *start_event_;
```

Comment:
these changes dont seem related to pip?

Reply:
This is to fix a bug with building the bindings of the GPU version of Marius. Came across it while testing the pip install and bundled it in with this pull request.

Reply:
Its generally better to have a separate PR for bug fixes -- makes it easier to track down later if you are looking at history to figure out when this was fixed and how etc.

Reply:
Make sense, I'll avoid doing this in future PRs